google/datalab/contrib/pipeline/_pipeline.py

```diff
@@ -104,6 +107,12 @@ def py(self):
     return Pipeline._imports + default_args + dag_definition + \
         task_definitions + up_steam_statements
 
+  @staticmethod
+  def _get_datetime_expr_str(date_string):
```

Comment:
I had it that way earlier, but the code wouldn't evaluate:

>>> datetime.datetime.strptime('2009-05-05T22:28:15Z', '%Y-%m-%dT%H:%M:%SZ').replace(tzinfo=timezone('UTC'))
datetime.datetime(2009, 5, 5, 22, 28, 15, tzinfo=<UTC>)
>>> datetime.datetime(2009, 5, 5, 22, 28, 15, tzinfo=<UTC>)
  File "<stdin>", line 1
    datetime.datetime(2009, 5, 5, 22, 28, 15, tzinfo=<UTC>)
                                                     ^
SyntaxError: invalid syntax